mod/item.php

```diff
@@ -393,48 +393,38 @@ function item_post(App $a) {
 
 	// Personal notes must never be altered to a forum post.
 	if ($posttype != Item::PT_PERSONAL_NOTE) {
-		$body = BBCode::performWithEscapedTags($body, ['noparse', 'pre', 'code', 'img'], function ($body) use ($profile_uid, $network, $str_contact_allow, &$inform, &$private_forum, &$private_id, &$only_to_forum, &$forum_contact) {
-			$tags = BBCode::getTags($body);
-
-			$tagged = [];
-
-			foreach ($tags as $tag) {
-				$tag_type = substr($tag, 0, 1);
-
-				if ($tag_type == Tag::TAG_CHARACTER[Tag::HASHTAG]) {
-					continue;
-				}
-
-				/* If we already tagged 'Robert Johnson', don't try and tag 'Robert'.
-				* Robert Johnson should be first in the $tags array
-				*/
-				foreach ($tagged as $nextTag) {
-					if (stristr($nextTag, $tag . ' ')) {
-						continue 2;
-					}
-				}
+		// Convert mentions in the body to a unified format
```

Comment:
This comment should either be in the method's documentation, or even better, in the name, like `unifyTagFormat`.

Reply:
I like writing comments into the code.

Reply:
And as a general rule, it's fine. However, if "converting mentions to a unified format" is an inherent feature of the `BBCode::setMentions` method, then I believe this should be made clear in the method's own documentation instead of hiding it next to one of the method's calls.

Reply:
I added some information to the function documentation.